tests/main/test_integration.py

```diff
@@ -93,13 +95,15 @@
 from ..fixtures import fixture_patched_xem_scripts_folder
 from ..fixtures import fixture_test_process_manager
 from ..fixtures_file_writer import GENERIC_START_RECORDING_COMMAND
+from ..fixtures_file_writer import WELL_DEF_24
 from ..helpers import confirm_queue_is_eventually_empty
 
 __fixtures__ = [
     fixture_fully_running_app_from_main_entrypoint,
     fixture_test_process_manager,
     fixture_patched_xem_scripts_folder,
     fixture_patched_firmware_folder,
+    WELL_DEF_24,
```

Comment:
note for future: we only need to put things in here if they are actual pytest fixtures---because `zimports` doesn't recognize them being used as 'function arguments' inside a test, and so deletes the import. an actual constant imported from a file shouldn't be an issue